### DISCUSSION TITLE
USB device not found starting docker container

### DISCUSSION THREAD

**Discussion by pmol200** [Koenkk/zigbee2mqtt#10819] [General]
My docker configuration is running various containers for Home Assistant.
One of the is zigbee2mqtt. Althought is doesn't seem to function, the error I get starting the container is:

Error: Error while opening serialport 'Error: Error: No such file or directory, cannot open /dev/ttyUSB0'

I checked  the USB devices with lsusb, this shows my zigbee stick: Bus 001 Device 003: ID 10c4:ea60 Silicon Labs CP210x UART Bridge
When running the command 'ls -l /dev/serial/by-id', I get the following output.
usb-Silicon_Labs_CP2102N_USB_to_UART_Bridge_Controller_924ede47fa93eb11a908aa5b3d98b6d1-if00-port0 -> ../../ttyUSB0
In my configuration.yaml file I use the following port to connect to the usb stick. 
  port: /dev/ttyUSB0

I tried rebooting my Pi and also using another usb port. This device worked fine until I reinstalled my Pi. It is only a few weeks old, so I don't think of a hardware error.

I also set permissions and ownership with the following commands:

chown <username> ttyUSB0
sudo usermod -a -G dialout $USER
sudo usermod -a -G plugdev $USER

The network it runs on is using the bridge configuration.

I have no clue what else I can do.
Anyone an idea of what could be the problem ?

**Comment by m0wlheld** [comment]
> I checked the USB devices with lsusb

I‘ll assume you checked the host, but did you pass /dev/ttyUSB0 to the container? Use docker argument `-v /dev/ttyUSB0:/dev/ttyUSB0` to do so.
